### DISCUSSION TITLE
Interval hourly chime

### DISCUSSION THREAD

**Discussion by 814d3** [joeycastillo/Sensor-Watch#237] [Ideas]
Hi all,

I really like the hourly chime, but I don't like the fact, that it rings for each of the 24h.
So wouldn't it be possible to add a feature (no additional watchface) to adjust an interval in which the hourly chime is activated? Then it could be auto off at night and auto on at daytime for example.

Greetings

**Comment by theorangerider** [comment]
I think I've found a way to hard code it (modify the check for the boolean return value in simple_clock_face_wants_background_task()), but it would be nicer if there was a way to integrate this into the preferences face. From what I can tell the intention is to use bitmasks for registers for preferences, so setting a "quiet time" doesn't quite fit in there.

**Comment by voloved** [comment]
I added logic in the settings to be able to choose a start and end time for the beeping. Due to the lack of spare bits in the global settings, it only allows for a few times, but it works well.

https://github.com/voloved/Sensor-Watch/commit/ceed39a8b98b8570a58bb320b821a5478eb394e9
